pallets/interest-accrual/src/tests.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 Centrifuge Foundation (centrifuge.io).
+// This file is part of Centrifuge chain project.
+
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+use sp_runtime::{
+	traits::{One, Zero},
+	FixedPointNumber,
+};
+
+use crate::{
+	mock::{Rate, Test as TestConfig},
+	Pallet,
+};
+
+#[test]
+fn test_rate_validation() {
```

Comment:
Should we add a ticket to add some more tests to this pallet? @branan

Reply:
We already have one. #849.

Most of the interesting code paths are covered by the loans tests anyway, but it's be good to have dedicated tests (and good to make loans less fragile by removing it validating internal/intermediate values from i-i)